support-frontend/assets/pages/supporter-plus-landing/twoStepPages/secondStepCheckout.tsx

```diff
@@ -66,6 +67,16 @@ export function SupporterPlusCheckout({
 	const { supportInternationalisationId } = countryGroups[countryGroupId];
 	const contributionType = useContributionsSelector(getContributionType);
 	const amount = useContributionsSelector(getUserSelectedAmount);
+	const amountBeforeTransactionCostCovered = useContributionsSelector((state) =>
+		getAmount(
+			state.page.checkoutForm.product.selectedAmounts,
+			state.page.checkoutForm.product.otherAmounts,
+			contributionType,
+			false,
+		),
+	);
+
+	// ToDo: clean this up?
```

Comment:
Could you add more context to this comment as I'm not sure someone would be able to work out what needs tidying?

Reply:
More of a note to self but maybe worth discussing. A previous PR / AB test introduced the concept of "amountBeforeAmendments" https://github.com/guardian/support-frontend/pull/5249 which feels similar? But is now unused

Reply:
Ah, I'd remove it for what you're doing, which is being explicit to what the amendments are.